### DISCUSSION TITLE
Why Python Selected as Xmigrate Programming Language ? #178

### DISCUSSION THREAD

**Discussion by iqlal** [vishnu-ks/Cloud-migration#179] [Q&A] [answered]
Why Python Selected as Xmigrate Programming Language ? #178

**Comment by iamvishnuks** [comment, marked as answer]
Python is the go to language for the end users of this application, DevOps and Cloud engineers. This applications is written in such a way that the application user would be able to go and fix any issues or extend the application to support additional features.
